src/plugins/ini/ini.c

```diff
@@ -38,6 +38,9 @@ static int iniCmpOrder (const void * a, const void * b);
 #define DEFAULT_DELIMITER '='
 #define DEFAULT_COMMENT_CHAR '#'
 
+char * const ININAME_DELIM_SPECIAL_MASK = "\"%s\"%c";
```

Comment:
```suggestion
char const * const ININAME_DELIM_SPECIAL_MASK = "\"%s\"%c";
```

You can also specify that the data (not only the pointer to the data) will not be changed here.